dashboard/src/modules/containers/MainLayout/index.jsx

```diff
@@ -1,28 +1,30 @@
-import React from 'react';
+import React from "react";
+import { Outlet } from "react-router-dom";
 import ToastComponent from "modules/components/ToastNotificationComponent";
 import { useSelector } from "react-redux";
-import { Page, Spinner } from '@patternfly/react-core';
+import { Spinner, Page, PageSection } from "@patternfly/react-core";
+import HeaderComponent from "modules/components/HeaderComponent";
 import NavbarDrawer from '../../components/NavbarDrawerComponent';
```

Comment:
The LGTM bot says you aren't using the `NavbarDrawer`, here.  (Did something get dropped during the merge?)

Reply:
`NavbarDrawer` has been changed to `HeaderComponent`